developers/integrations/index.md

```diff
@@ -6,25 +6,32 @@ image: og/integrations/home.jpg
 
 Weaviate's integration ecosystem enables developers to build various applications leveraging Weaviate and another technology.
 
-You can find all notebooks and examples on [Weaviate Recipes](https://github.com/weaviate/recipes)!
+All the notebooks and code examples are on [Weaviate Recipes](https://github.com/weaviate/recipes)!
+
+<img
+    src={require('./ecosystem.png').default}
+    alt="alt"
+    style={{ maxWidth: "60%", display: "block", marginLeft: "auto", marginRight: "auto"}}
+/>
+
 
 ## About the Categories
-The ecosystem is broken out by 5 categories:
+The ecosystem is broken into five categories:
```

Comment:
Nice catch on five instead of 5!

Consider one of these in case it changes later: 
The ecosystem is divided into these categories:
The integrations are grouped into these categories: